src/Internal.AspNetCore.Sdk/build/Common.targets

```diff
@@ -15,6 +15,16 @@ For single-tfm projects, this will be imported from build/Internal.AspNetCore.Sd
     <IncludeSymbols Condition="'$(NuspecFile)'!=''">false</IncludeSymbols>
   </PropertyGroup>
 
+  <PropertyGroup Condition=" '$(IncludeSymbols)' != 'false' ">
+    <!-- Include .pdb files in regular .nupkg files. -->
+    <AllowedOutputExtensionsInPackageBuildOutputFolder>$(AllowedOutputExtensionsInPackageBuildOutputFolder);.pdb</AllowedOutputExtensionsInPackageBuildOutputFolder>
+  </PropertyGroup>
+
+  <Target Name="_EnsurePdbsInNupkgArePortable" BeforeTargets="GenerateNuspec" Condition=" '$(IsPackable)' != 'false' ">
+    <Error Text="Symbols included in a nupkg package should be portable. Set DebugType=portable or IncludeSymbols=false."
```

Comment:
Set DebugType=portable ~~or~~and IncludeSymbols=false

Reply:
Maybe I should clarify. What I meant here was `Fix by setting DebugType=portable or disable inclusion of symbols by setting IncludeSymbols=false`

Reply:
Maybe split this in to two? 

> Symbols were set to be included but the project was built using `DebugType = full`. Set `DebugType = portable` to include portable pdbs required for symbol publishing or stop including symbols by setting `IncludeSymbols=false`

Reply:
that's much better. I'll update the error.